### PR TITLE
ci: add test for executable markdown

### PR DESCRIPTION
**Pull request by abitrolly**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -28,6 +28,19 @@ jobs:
...
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - macos-latest
+          - ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: teaxyz/setup@v0
+      - run: tea .
```
Comment (mxcl) [inline .github/workflows/ci.yml:42]: Needs to be `tea run .` or it won’t run with the current source code. Our `run` target runs `deno` with the working directory sources.

**Review by mxcl — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (mxcl) [reply .github/workflows/ci.yml:42]: Also you may need to rebase since I just fixed the “cannot write” error.

Comment (abitrolly) [reply .github/workflows/ci.yml:42]: But documentation says `tea .` should work  - https://github.com/teaxyz/cli#executable-markdown
```sh
$ tea .  # interprets `# Getting Started`, could also be `tea ./README.md`
```

Comment (abitrolly) [reply .github/workflows/ci.yml:42]: Rebased.

Comment (mxcl) [reply .github/workflows/ci.yml:42]: `run` is an exe/md target in our `README`.

**Review by mxcl — APPROVED** [review]

**Pull request merged**